doc/api/jstp.md

```diff
@@ -151,15 +151,15 @@ Select a record or multiple records in the category using provided jsql.
 
 ###### Call arguments
 
-* `category` [`<string>`][]
-* `jsql` [`<Object[]>`][object]
+- `category` [`<string>`][]
+- `jsql` [`<Object[]>`][object]
 
 ###### Callback arguments
 
-* `data` [`<Object[]>`][object], fetched data
+- `data` [`<Object[]>`][object], fetched data
 
 [errors]: https://github.com/metarhia/globalstorage/blob/master/lib/errors.js
-[`<Object>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
+[`<object>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
```

Comment:
I think you can remove this link and instead use `[object]` in all the related places.
```suggestion
```